src/ckeditor.tsx

```diff
@@ -422,44 +443,27 @@ export default class CKEditor<TEditor extends Editor> extends React.Component<Pr
 
 	public static override contextType = ContextWatchdogContext;
 
-	// Properties definition.
-	public static propTypes = {
-		editor: PropTypes.func.isRequired as unknown as Validator<{ create( ...args: any ): Promise<any> }>,
-		data: PropTypes.any,
-		attributes: PropTypes.any,
-		sourceElements: PropTypes.object,
-		config: PropTypes.object,
-		watchdogConfig: PropTypes.object,
-		onChange: PropTypes.func,
-		onReady: PropTypes.func,
-		onFocus: PropTypes.func,
-		onBlur: PropTypes.func,
-		onAddRoot: PropTypes.func,
-		onRemoveRoot: PropTypes.func,
-		onError: PropTypes.func,
-		disabled: PropTypes.bool,
-		id: PropTypes.any
-	};
```

Comment:
Why you removed the run-time properties validation?

Reply:
For me `prop-types` should be removed since there is almost no value in using it. But okay, I will restore it.